src/jgikbase/idmapping/service/mapper_service.py

```diff
@@ -66,6 +69,25 @@ def _objids_to_jsonable(oids: Set[ObjectID]):
                   key=itemgetter('namespace', 'id'))
 
 
+def _check_id(id_, name: str) -> str:
```

Comment:
This could be pulled out into the `arg_check` module and called something like "not_blank"

Reply:
It winds up getting removed in the switch to bulk endpoints, so moot.